known-issues(mbi): refacto known issues + add procedure for MBI bigint columns (#1440)
centreon/centreon-documentation: versioned_docs/version-22.04/resources/known-issues.md

necessary and are forever improving our software in order to solve any
issues for future releases.

# Centreon Web

## You have reached the maximum of id into `centreon_storage.index_data`

### Workaround

Play the following request into MariaDB :

In your monitoring database :
```sql
ALTER TABLE index_data MODIFY id bigint unsigned AUTO_INCREMENT;
ALTER TABLE metrics MODIFY index_id bigint unsigned;
```

In your configuration database :
```sql
ALTER TABLE ods_view_details MODIFY index_id bigint unsigned;
ALTER TABLE virtual_metrics MODIFY index_id bigint unsigned;
```

> > :warning: **Depending on your metrics volumes, this operation could be more or less longer.**

Reviewer: ```suggestion
> Depending on your metrics volumes, this operation could be more or less long.
```

## Autologin does not work with some pages

### Description

The autologin is cuurently not managed on the following pages :
* *Monitoring > Resources Status*
* *Configuration > Hosts > Discovery*
* *Configuration > Business Activity > Business Views*
* *Configuration > Business Activity > Business Activity*

### Workaround

There is currently no workaround.

## The content of the pages is not translated according to the language of the user

### Workaround

You must install the languages on your operating system with the following command:
```shell
yum install -y glibc-all-langpacks
```

Then restart PHP using the following command:
```shell
systemctl restart php-fpm
```

# Centreon MBI

## The **Report Parameters** tab of a job is empty (**Reporting > Monitoring Business Intelligence > Jobs**)

### Workaround

* Go to the **Configuration** tab of the job
* Select a different report design from the **Report design** list
* Select the original report design
* Go back to the **Report Parameters** tab

> ## Vous avez atteint le nombre maximum d'id pour les colonnes `servicemetric_id`

Author: ```suggestion
## You have reached the maximum of id for `servicemetric_id` columns
```

### Description

> If you have a very large infrastructureSi vous possédez une très large infrastructure, it is possible that the `servicemetric_id` column size has been reached.

Author: ```suggestion
If you have a very large infrastructure, it is possible that the `servicemetric_id` column size has been reached.
```
